src/main/kotlin/com/terraformation/backend/accelerator/api/ParticipantProjectSpeciesController.kt

```diff
@@ -90,6 +101,30 @@ class ParticipantProjectSpeciesController(
     return GetParticipantProjectSpeciesResponsePayload(ParticipantProjectSpeciesPayload(model))
   }
 
+  @ApiResponse(
+      responseCode = "200",
+      description = "The file was successfully retrieved.",
+      content =
+          [
+              Content(
+                  schema = Schema(type = "string", format = "binary"),
+                  mediaType = MediaType.ALL_VALUE)])
+  @GetMapping("/projects/{projectId}/species/snapshots/{deliverableId}")
+  @Operation(summary = "Creates a new participant project species entry.")
+  @Produces
+  fun getParticipantProjectSpeciesSnapshot(
+      @PathVariable projectId: ProjectId,
+      @PathVariable deliverableId: DeliverableId
+  ): ResponseEntity<InputStreamResource> {
+    return try {
+      participantProjectSpeciesService
+          .readSubmissionSnapshotFile(projectId, deliverableId)
+          .toResponseEntity { contentDisposition = ContentDisposition.attachment().build() }
```

Comment:
Should this header include a filename?

Reply:
The way this is set up in the frontend is using the pre-existing "download CSV" modal which allows the user to enter a filename.